### PR TITLE
[CELEBORN-1250][FOLLOWUP] Fix license issues

### PR DESCRIPTION
**Pull request by FMX**
### What changes were proposed in this pull request?
Fix license issues.

### Does this PR introduce _any_ user-facing change?
NO.

### How was this patch tested?
GA.

### CODE REVIEW

File: LICENSE-binary [hunk 1]
```diff
@@ -284,21 +284,21 @@ for text of these licenses.
...
+See licenses/LICENSE-CDDL.txt for details.
 javax.servlet:javax.servlet-api
 
 
 MIT License
 ------------
-See license/LISENCE-jdktools.txt for details.
+See licenses/LICENSE-jdktools.txt for details.
 com.github.olivergondza:maven-jdk-tools-wrapper
 See license/LICENSE-slf4j.txt for details.
```
Comment (SteNicholas) [inline LICENSE-binary:303]:
```suggestion
See licenses/LICENSE-slf4j.txt for details.
```

File: client-mr/mr-shaded/src/main/resources/META-INF/LICENSE [hunk 2]
```diff
@@ -247,7 +247,7 @@ org.scala-lang:scala-library
 
 BSD
 ------------
-See license/LICENSE-protobuf.txt for details.
+See licenses/LICENSE-protobuf.txt for details.
 com.google.protobuf:protobuf-java
 See license/LICENSE-zstd-jni.txt for details.
```
Comment (SteNicholas) [inline client-mr/mr-shaded/src/main/resources/META-INF/LICENSE:252]:
```suggestion
See licenses/LICENSE-zstd-jni.txt for details.
```

**Review by SteNicholas — APPROVED** [review]
LGTM. cc @pan3793, @cfmcgrady.

File: LICENSE-binary [hunk 3]
```diff
@@ -284,23 +284,23 @@ for text of these licenses.
...
 MIT License
 ------------
-See license/LISENCE-jdktools.txt for details.
+See licenses/LICENSE-jdktools.txt for details.
 com.github.olivergondza:maven-jdk-tools-wrapper
-See license/LICENSE-slf4j.txt for details.
+See licenses/LICENSE-slf4j.txt for details.
 org.slf4j:jul-to-slf4j
 org.slf4j:slf4j-api
 ------------
```
Comment (cfmcgrady) [inline LICENSE-binary:306]: L307
```diff
- See license/LICENSE-javassist.txt for detail.
+ See licenses/LICENSE-javassist.txt for detail.
```

**Review by RexXiong — APPROVED** [review]
LGTM, thanks!

**Comment by FMX** [comment]
Merged into branch-0.4(v0.4.0)

**Pull request closed without merging**